src/iop/denoiseprofile.c

```diff
@@ -3103,6 +3344,11 @@ void gui_init(dt_iop_module_t *self)
   dt_bauhaus_widget_set_label(g->strength, NULL, _("strength"));
   dt_bauhaus_combobox_add(g->mode, _("non-local means"));
   dt_bauhaus_combobox_add(g->mode, _("wavelets"));
+  gboolean compute_variance = dt_conf_get_bool("plugins/darkroom/denoiseprofile/show_compute_variance_mode");
```

Comment:
const ...

And you need an entry data/darktableconfig.xml.in as others hidden prefs, this at least serve as documentation.

Reply:
Thank you for the review @TurboGit :-)
I have now made these changes